src/scripts/main.js

```diff
@@ -1,3 +1,101 @@
 'use strict';
 
-// write code here
+const head = document.querySelector('thead');
+const body = document.querySelector('tbody');
+const bodychildren = [...body.children];
+
+function transformationNumber(element) {
+  const array = element.children[3].textContent.slice(1).split(',');
+
+  return +(array[0] + array[1]);
+}
+
+head.addEventListener('click', even => {
+  if (even.target.textContent === 'Age') {
+    bodychildren.sort((one, two) => {
+      return +one.children[2].textContent - +two.children[2].textContent;
+    });
+
+    if (even.target.dataset.clicked === undefined) {
+      for (const child of bodychildren) {
+        body.append(child);
+      }
+      even.target.dataset.clicked = true;
+
+      return;
+    }
+
+    if (even.target.dataset.clicked === 'true') {
+      for (const child of bodychildren) {
+        body.prepend(child);
+      }
+      even.target.removeAttribute('data-clicked');
+    }
```

Comment:
why did you need this code?

Reply:
These conditions add elements to the column. The first condition adds in ascending order, the other in descending order.